README.md

```diff
@@ -168,6 +168,9 @@ You can define the next environment variables to change values from NGINX and PH
 | PHP8   | post_max_size           | 8M            | Max size of post data allowed. This setting also affects file upload. To upload large files, this value must be larger than upload_max_filesize. Generally speaking, memory_limit should be larger than post_max_size.                                 |
 | PHP8   | upload_max_filesize     | 2M            | Maximum size of an uploaded file.                                                                                                                                                                                                                      |
 | PHP8   | zlib_output_compression | On            | Whether to transparently compress pages. If this option is set to "On" in php.ini or the Apache configuration, pages are compressed if the browser sends an "Accept-Encoding: gzip" or "deflate" header.                                               |
+| PHP8   | date_timezone | UTC            | Set the Timezone of PHP                                               |
```

Comment:
Try to align the spaces like the others, add more spaces between` | date_timezone `and` | UTC `to be aligned with the rest of the document.